watermarker/src/commonMain/kotlin/Watermarker.kt

```diff
@@ -147,16 +147,25 @@ open class Watermarker {
      * Returns all watermarks in [text].
      *
      * When [squash] is true: watermarks with the same content are merged.
+     * When [singleWatermark] is true: only the most frequent watermark is returned.
      */
     fun textGetWatermarks(
```

Comment:
Could you please update and enhance the documentation? Currently, 3 possible usages are described here (for Kotlin, Java, and JavaScript):
https://fraunhoferisst.github.io/TREND/docs/usage/watermarker/

With the new `singleWatermark` option, the code can be enhanced and updated. It should be checked if lines like `check(extractedWatermarks.size == 1)` can be removed because they are no longer necessary.